deploy/cap/syslog.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "pry"
+
+namespace :syslog do
+  def journalctl_cmd(initial: [:sudo, "/bin/journalctl"])
+    fetch(:systemd_services).reduce(initial) {|memo, obj| memo + ["-u", obj] }
```

Comment:
A reduce here seems a little exotic. This is `a + b` or `[a + b].flatten`, right?

Reply:
This concatenates `["-u", "#{service}"]` to the array for each item in `:systemd_services`

Reply:
Oh, I see. This seems more of a map than a reduce to me. What about this?

```ruby
def journalctl_cmd(journalctl: [: sudo, "/bin/journalctl"])
  units = fetch(:system_services).map {|unit| ['-u', unit] }.flatten
  journalctl + units
end
```